src/index.html

```diff
@@ -1,13 +1,464 @@
+                      </path>
+                      <path
+                        d="M12.0001 6.84465C9.15753 6.84465 6.84473 9.15727 6.84473 12C6.84473 14.8425 9.15753 17.1553 12.0001 17.1553C14.8428 17.1553 17.1554 14.8425 17.1554 12C17.1554 9.15727 14.8428 6.84465 12.0001 6.84465Z">
+                      </path>
+                      <path
+                        d="M18.7191 4.51408C18.4258 4.51408 18.187 4.75285 18.187 5.04619C18.187 5.33952 18.4258 5.57829 18.7191 5.57829C19.0126 5.57829 19.2514 5.3397 19.2514 5.04619C19.2514 4.75267 19.0126 4.51408 18.7191 4.51408Z">
+                      </path>
+                    </g>
+                    <defs>
+                      <clippath id="clip0_12665_559">
+                        <rect width="24" height="24"></rect>
+                      </clippath>
+                    </defs>
+                  </svg>
+                </a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="footer__schedule">
+            <h4 class="footer__schedule-title">Графік роботи</h4>
+
+            <ul class="schedule">
+              <li class="schedule__text">
+                <time class="schedule__text--bold">12:00 — 20:00</time>
+                середа
+              </li>
+
+              <li class="schedule__text">
+                <time class="schedule__text--bold">12:00 — 20:00</time>
+                четвер
+              </li>
+
+              <li class="schedule__text">
+                <time class="schedule__text--bold">12:00 — 20:00</time>
+                п’ятниця
+              </li>
+
+              <li class="schedule__text">
+                <time class="schedule__text--bold">11:00 — 19:00</time>
+                субота
+              </li>
+
+              <li class="schedule__text">
+                <time class="schedule__text--bold">11:00 — 19:00</time>
+                неділя
+              </li>
+            </ul>
+
+            <p class="schedule__on-weekend">
+              <strong>Вихідні:</strong> понеділок,
+              вівторок <br> Каса припиняє роботу за 1 годину до закриття музею
+            </p>
+          </div>
+
+          <div class="contact footer__contact">
+            <h4 class="contact__title footer__contact-title">Контакти:</h4>
+
+            <ul class="contact__list">
+              <li class="contact__item">
+                <a
+                  href="https://goo.gl/maps/pwqfvXxsF41ewwxE6"
+                  class="contact__link"
+                  target="_blank"
+                >вул. М. Грушевського, 6; Київ,
+                  01001
+                </a>
```

Comment:
```suggestion
                <a
                  href="https://goo.gl/maps/pwqfvXxsF41ewwxE6"
                  class="contact__link"
                  target="_blank"
                >
                  вул. М. Грушевського, 6; Київ,
                  01001
                </a>
```

Reply:
.I apologize for such obvious mistakes, I was just a little unfocused. I think it's okay now)